### PR TITLE
Feature 69 product calculated price

### PR DESCRIPTION
**Pull request by dimitrov93**

### CODE REVIEW

File: src/components/Form/ProductCreateAndEditForm.vue [hunk 1]
```diff
@@ -94,16 +86,44 @@
           </v-btn>
         </div>
 
-        <div class="d-flex flex-column mt-4">
-          <p v-if="props.productInfo.resourcesContent?.length > 0">
+        <v-text-field
+          v-if="props.productInfo.resourcesContent?.length > 0"
```
Comment (VladoKat) [inline src/components/Form/ProductCreateAndEditForm.vue:90]: I see we use `"props.productInfo.resourcesContent?.length > 0"` in every html tag below. Can we wrap all of them in a single section (maybe div) and then we would write this only once.

File: src/views/Products/ProductCreate.vue [hunk 2]
```diff
@@ -50,6 +50,7 @@ const createProduct = async () => {
     snackbarProvider.showSuccessSnackbar("Successfully added product!");
     return res;
   } catch (error) {
+    console.log(error);
```
Comment (VladoKat) [inline src/views/Products/ProductCreate.vue:53]: I guess we don't want this console.log?

File: src/views/Products/ProductsDetailsById.vue [hunk 3]
```diff
@@ -46,11 +46,25 @@
...
         </div>
 
-        <strong>Part of Sale: </strong>
-        <part-of-product
-          :partOfProduct="currentProductInfo.partOfSale"
-          routerPreFix="sales"
-        />
+        <div>
+          <strong>Part of Sale: </strong>
+          <part-of-product
```
Comment (VladoKat) [inline src/views/Products/ProductsDetailsById.vue:51]: This name looks wrong but I guess this is what it was named initially. Shouldn't it be part-of-sale or I am missing something

File: src/views/Products/ProductsDetailsById.vue [hunk 4]
```diff
@@ -119,6 +133,7 @@ const currentProductId = route.params.productId;
 const currentProductInfo = computed(
   () => store.getters["products/allProducts"]
 ).value.find((product) => product.id === currentProductId);
+console.log(currentProductInfo);
```
Comment (VladoKat) [inline src/views/Products/ProductsDetailsById.vue:136]: Another wild console.log appears

File: tests/Products/details-product.spec.js [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+  await navigateToPage(page, expect, "products");
+  await createGlobalVariables(page);
+  await page.goto(`products/${PRODUCT_ID}`);
+});
+
+test.afterEach(async ({ page }) => {
+  await page.close();
+});
+
+test("Catalog fields are visible", async ({ page }) => {
```
Comment (VladoKat) [inline tests/Products/details-product.spec.js:18]:
```suggestion
test("Product fields are visible", async ({ page }) => {
```

**Review by VladoKat — CHANGES_REQUESTED** [review]
Minor tweaks required. Tests look way better. Way to go! 🚀

File: tests/Products/details-product.spec.js (hunk 5 above)
Comment (dimitrov93) [reply tests/Products/details-product.spec.js:18]: updated

File: src/views/Products/ProductsDetailsById.vue (hunk 3 above)
Comment (dimitrov93) [reply src/views/Products/ProductsDetailsById.vue:51]: yes, in the way we use it is wrong, but its a component we are using twice, once for contentOf and once for partOfSale

File: src/components/Form/ProductCreateAndEditForm.vue (hunk 1 above)
Comment (dimitrov93) [reply src/components/Form/ProductCreateAndEditForm.vue:90]: exported to a component

File: src/components/SectionInfoPanel.vue
Comment (VladoKat) [inline src/components/SectionInfoPanel.vue:1]: Let's rename it to ProductContentsInfoPanel?

**Review by VladoKat — CHANGES_REQUESTED** [review]

File: src/components/SectionInfoPanel.vue
Comment (dimitrov93) [reply src/components/SectionInfoPanel.vue:1]: updated

**Review by VladoKat — APPROVED** [review]

**Pull request merged**